src/ctia/observable/routes.clj

```diff
@@ -25,7 +27,10 @@
             (s/optional-key :sort_by)
             (describe (s/enum :id) "Sort result on a field")))
 
-(defroutes observable-routes
+(s/defn observable-routes [{{:keys [get-in-config]} :ConfigService
+                            {:keys [read-store]} :StoreService
+                            :as services} :- APIHandlerServices]
```

Comment:
```suggestion
                            {:keys [read-store]} :StoreService} :- APIHandlerServices]
```

Reply:
Thanks, I'm juggling lots of these `_services_` vs `services` vs nothing. If you'll indulge me, using `_services_` is a little easier for me to manage for (as-of-now) unused service args.